app/src/org/commcare/preferences/CommCareServerPreferences.java

```diff
@@ -96,6 +96,10 @@ public static String getSupportEmailAddress() {
         return getServerProperty(PREFS_SUPPORT_ADDRESS_KEY, CommCareApplication.instance().getString(R.string.support_email_address_default)) ;
     }
 
+    public static String getFormRecordKey() {
```

Comment:
Yeah. I agree as well. I am going to proceed with moving them both to Developer Prefs.